AIToolbox/experiment_save/local_results_save.py

```diff
@@ -160,7 +160,7 @@ def save_experiment_results(self, result_package, project_name, experiment_name,
                                        'training_history': training_history}
 
         if save_true_pred_labels:
-            exp_results_hyperparam_dict = {'y_true': result_package.y_true.tolist(), 'y_predicted': result_package.y_predicted.tolist(),
+            exp_results_hyperparam_dict = {'y_true': result_package.y_true, 'y_predicted': result_package.y_predicted,
```

Comment:
removed .tolist()... be on guard if this brakes something in the result saving pipeline down the line. Based on the tests I executed everything still works fine.

To list was removed to enable saving both lists as well as dicts in case you used MultipleResultPackageWrapper